src/jsx/index.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path="../../node_modules/types-for-adobe/shared/PlugPlugExternalObject.ts" />
```

Comment:
```suggestion
```

Let's put this in `console.ts` where its immediately used.

Reply:
I agree